### PR TITLE
Reconcile loggers

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
This pull request fixes a regression introduced with #567 and (slightly) tries to improve the overall approach to logging. There is still material for three more pull requests.

### CODE REVIEW

**Review by dacbd — APPROVED** [review]

File: iterative/utils/logger_test.go [hunk 1]
```diff
@@ -16,7 +17,8 @@ func TestState(t *testing.T) {
 		},
 	})
 
-	logger := TpiLogger(d)
+	logger := logrus.WithFields(logrus.Fields{"d": d})
+	logrus.SetFormatter(&TpiFormatter{})
```
Comment (casperdcl) [inline iterative/utils/logger_test.go:21]: thins kind of thing just begging for a helper function seeing as it's used a lot?

**Comment by casperdcl** [comment]
> There is still material for three more pull requests

follow-up issue(s)? :D

File: iterative/utils/logger_test.go (hunk 1 above)
Comment (0x2b3bfa0) [reply iterative/utils/logger_test.go:21]: Not worth the hassle: it's used often in “tests” but only once in the code.

**Comment by 0x2b3bfa0** [comment]
Or even pull requests: will depend on the new branch.

**Pull request merged**